components/brave_rewards/resources/page/components/provider_redirect_modal.tsx

```diff
@@ -187,12 +180,13 @@ export function ProviderRedirectModal () {
       return (
         <ModalRedirect
           id={'redirect-modal-error'}
-          errorText={[getString('redirectModalError')]}
-          buttonText={getString('processingRequestButton')}
-          titleText={getString('processingRequest')}
+          errorText={[getString('redirectModalUnableToCompleteRequest')]}
+          errorTextLink={'https://community.brave.com'}
```

Comment:
Is this one already in `shared/lib/rewards_urls.ts`?

Reply:
Yes, it is. I'll replace them through the file.

Reply:
https://github.com/brave/brave-core/pull/21863/commits/485dd84d7a96994e42f712a486bcf1929a7c4c47